src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form-client/nfs-form-client.component.html

```diff
@@ -37,7 +37,7 @@
                        placeholder="192.168.0.10, 192.168.1.0/8">
                 <span class="invalid-feedback">
                   <span *ngIf="showError(index, 'addresses', formDir, 'required')"
-                        i18n>Required field</span>
+                        i18n>This field is required.</span>
```

Comment:
Just to add more fun: [Patterfly suggests](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/forms) **"This is a required field"**. Cannot grasp the nuances of each one...

Is there a way to have this as a re-usable string, so we don't need to specify it every time?

Reply:
We could introduce a constant in `src/app/shared/constants/app.constants.ts` after we agreed on one sentence.

Design guides suggesting `This field is required`:
* [Lightning Design System](https://www.lightningdesignsystem.com/components/input/#Error) (Salesforce)
* [Project Clarity](https://clarity.design/documentation/input)
* [VeeValidate](https://logaretm.github.io/vee-validate/guide/rules.html#importing-the-rules)
* [Google Trends](https://trends.google.com/trends/explore?date=today%205-y&q=This%20field%20is%20required,This%20is%20a%20required%20field)
* Google search: ~43.600.000 hits

Design guides suggesting `This is a required field`:
* [Patternfly](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/forms)
* Google search: ~4.050.000.000 hits